docs/components/dialogs.md

```diff
@@ -0,0 +1,471 @@
+<!--docs:
+title: "Dialogs"
+layout: detail
+section: components
+excerpt: "Dialogs are modal windows that require interaction."
+iconId: 
+path: /catalog/dialog/
+-->
+
+
+# Dialogs
+
+Dialogs inform users about a task and can contain critical information, require decisions, or involve multiple tasks.
+
+There are four types of dialogs:
+
+1. [Alert](#alert-dialog)
+1. [Simple](#simple-dialog)
+1. [Confirmation](#confirmation-dialog)
+1. [Full-screen](#full-screen-dialog)
```

Comment:
The list should be in-line:

"There are four types of dialogs: 1\. [Alert](#alert-dialog), 2\. [Simple](#simple-dialog), 3\. [Confirmation](#confirmation-dialog), 4\. [Full-screen](#full-screen-dialog)

Reply:
done